vignettes/GenomicFiles.Rmd

```diff
@@ -0,0 +1,687 @@
+---
+title: "Introduction to GenomicFiles"
+authors:
+  - name: "Valerie Obenchain"
+  - name: "Michael Love"
+  - name: "Martin Morgan"
+  - name: "Sonali Kumari"
+    affiliation: "Vignette translation from Sweave to Rmarkdown / HTML"
+date: "Last modified: October 2014; Compiled: `r format(Sys.time(), '%B %d , %Y')`"
+
+vignette: >
+  %\VignetteIndexEntry{Introduction to GenomicFiles}
+  %\VignetteDepends{GenomicAlignments, RNAseqData.HNRNPC.bam.chr14}
+  %\VignetteEncoding{UTF-8}
+  %\VignetteEngine{knitr::rmarkdown}
+output:
+  BiocStyle::html_document
+---
+
+```{r echo=FALSE, include=FALSE}
+if (!require("BiocManager", quietly = TRUE))
+    install.packages("BiocManager")
+BiocManager::install("BiocStyle", force = TRUE)
+```
+
+```{r include=FALSE}
+library(knitr)
+opts_chunk$set(tidy=FALSE)
+```
+
+```{r include=FALSE}
+library(BiocStyle)
+```
+
+# Introduction
+
+This vignette illustrates how to use the `r Biocpkg("GenomicFiles")`
+package for distributed computing across files. The functions in
+`GenomicFiles` manipulate and combine data subsets via two user-supplied
+functions, MAP and REDUCE. These are similar in spirit to `Map` and
+`Reduce` in base R. Together they provide a flexible interface to
+extract, manipulate, and combine data. Both functions are executed in
+the distributed step, which means results are combined on a single
+worker, not across workers.
+
+We assume the reader has some previous experience with R and with basic
+manipulation of ranges objects such as `GRanges` and `GAlignments` and
+file classes such as `BamFile` and `BigWigFile`. See the vignettes and
+documentation in `r Biocpkg("GenomicRanges")`, `r Biocpkg("GenomicAlignments")`, `r Biocpkg ("GenomicAlignments")`, `r Biocpkg("Rsamtools")` and `r Biocpkg("rtracklayer")` for an introduction to these classes.
+
+The GenomicFiles package is available at bioconductor.org and can be downloaded via `BiocManager::install`.
+```{r install, eval=FALSE}
+if (!require("BiocManager"))
+    install.packages("BiocManager")
+BiocManager::install("GenomicFiles")
+```
+
+# Quick Start
+
+GenomicFiles offers functions for the parallel extraction and combination of data
+subsets. A user-defined MAP function extracts and manipulates data while an optional REDUCE
+function consolidates the output of MAP.
+
+```{r quick_start-load, message=FALSE}
+library(GenomicFiles)
+
```

Comment:
Remove blank lines at the beginning or end of code chunks.

Reply:
> Hi @sonali8434 , your .Rmd is starting to look good and I am able to knit it. I am going to ask you to make a few more changes before I compare the new and old documents.
> 
> Good catch on the spelling mistake. Usually, if it is simple, it is okay to make. For others that are more complicated or where there are several, I usually advise the contributor to bring it up with reviewer/maintainer.

Hi @jwokaty I have done all the changes as suggested you can now continue with the review process.

Reply:
> Hi @sonali8434 , your .Rmd is starting to look good and I am able to knit it. I am going to ask you to make a few more changes before I compare the new and old documents.
> 
> Good catch on the spelling mistake. Usually, if it is simple, it is okay to make. For others that are more complicated or where there are several, I usually advise the contributor to bring it up with reviewer/maintainer.

Hi @jwokaty I have done all the changes as suggested you can now continue with the review process.